server/database/index.js

```diff
@@ -9,7 +9,7 @@ const pool = new Pool({
     port: process.env.DATABASE_PORT
 })
 
-const getTime = (request, response) => {
+const checkConnection = (request, response) => {
```

Comment:
Good job renaming this! It's much more explicit now!

Reply:
Thanks!